src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -174,7 +177,10 @@ public void parse_withExtraWhitespaces_success() throws Exception {
 
     @Test
     public void parse_configFolderOnly_success() throws Exception {
-        String input = new InputBuilder().addConfig(CONFIG_FOLDER_ABSOLUTE).build();
+        String input = new InputBuilder()
+                .addConfig(CONFIG_FOLDER_ABSOLUTE)
+                .addTimezone(DEFAULT_TIMEZONE)
```

Comment:
is it necessary to add the timezone into the input builder as these tests are specifically for certain flags? i.e. for this test, its testing for the config flag